framework/auth/oauth_scopes.py

```diff
@@ -124,7 +124,7 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ, CoreScopes.ALWAYS_PUBLIC, )
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
     USERS_CREATE = USERS_READ + (CoreScopes.USERS_CREATE, )
-    USERS_EMAIL = (CoreScopes.USERS_EMAIL, )
+    USER_EMAIL_READ =  (CoreScopes.USERS_READ, CoreScopes.USER_EMAIL_READ, )
```

Comment:
This was also confusing to me, after some discussions w/ @mfraezz its a bit clearer. 

The `scope` helper was added and the `ComposedScope`'s w/ `ALWAYS_PUBLIC` were left over from the first implementation. (I'll clean this up when I merge the PR)

This functionality basically allows any view marked `ALWAYS_PUBLIC` (very few, where making a specific scope was not warranted) to be called as long as the user has any valid Bearer token w/ scope.

The thing to remember (and the naming certainly is not helping) is `CoreScopes` are attached to Views. Where `ComposedScopes` are merely tuples of `CoreScopes` and only attached to the `public_scopes` below.

I like to think of these as each serves the other side of the authorization spectrum. Where the `public_scopes` are granted by the user at the beginning during the OAauth2 flow, and the `CoreScopes` lock down the views, so when a view is called the system verifies it's list of `CoreScopes` against the capabilities granted to the `public_scopes` -> `ComposedScope` on behalf of CAS.